baselines/teams.md

```diff
@@ -55,144 +56,175 @@ document. The United States Government has adapted selections of these
+- _Rationale:_ TODO add rationale.
+- _Last modified:_ July 2023
+- Custom meeting policies MAY be created that allow more flexibility for specific users.
 
-### 2.2.1 Policy
+#### MS.TEAMS.2.2v1
+Internal users SHOULD be admitted automatically in the Global (Org-wide default) meeting policy.
+- _Rationale:_ TODO add rationale.
+- _Last modified:_ July 2023
+- Custom meeting policies MAY be created that allow more flexibility for specific users.
 
-- Anonymous users SHALL NOT be enabled to start meetings in the Global
-(Org-wide default) meeting policy or in custom meeting policies if any
-exist.
+#### MS.TEAMS.2.3v1
+B2B guest users MAY be admitted automatically in the Global (Org-wide default) meeting policy.
+- _Rationale:_ TODO add rationale.
+- _Last modified:_ July 2023
+- Custom meeting policies MAY be created that allow more flexibility for specific users.
 
-### 2.2.2 Resources
 
-- [Meeting policy settings - Participants & guests \| Microsoft
-Docs](https://docs.microsoft.com/en-us/microsoftteams/meeting-policies-participants-and-guests)
 
-### 2.2.3 License Requirements
 
+### Resources
+- [Meeting policy settings - Participants & guests \| Microsoft
+  Docs](https://docs.microsoft.com/en-us/microsoftteams/meeting-policies-participants-and-guests)
+
+### License Requirements
 - N/A
 
-### 2.2.4 Implementation
+### Implementation
+All the settings in this section are configured in the [**Microsoft Teams admin
+    center**](https://admin.teams.microsoft.com).
 
-To configure settings for anonymous users:
+#### MS.TEAMS.2.1v1, instructions:
 
 1.  Sign in to the [**Microsoft Teams admin
     center**](https://admin.teams.microsoft.com).
+2.  Select **Meetings >** **Meeting** **policies**.
+
+3.  Select the **Global (Org-wide default)** policy.
+
+4.  Under the **Meeting join & lobby** section **,** ensure
+    **Who can bypass the lobby** is *not* set to **Everyone**. Bypassing the lobby should be set to **People in my org**, though other options may be used if needed. 
+
+5.  In the same section, set **People dialing in can bypass the lobby** to
+    **Off**.
 
-2.  Select **Meetings -\>** **Meeting policies**.
+#### MS.TEAMS.2.2v1, instructions:
+
+1.  Sign in to the [**Microsoft Teams admin
+    center**](https://admin.teams.microsoft.com).
+2.  Select **Meetings >** **Meeting** **policies**.
 
 3.  Select the **Global (Org-wide default)** policy.
 
-4.  Under the **Participants & guests** section**,** set **Let anonymous
-    people start a meeting** to **Off.**
+4.  Under the **Meeting join & lobby** section **,** ensure
+    **Who can bypass the lobby** is set to **People in my org**.
 
-5.  If custom policies have been created, repeat these steps for each
-    policy, selecting the appropriate policy in step 3.
+5.  In the same section, set **People dialing in can bypass the lobby** to
+    **Off**.
 
-## 2.3 Automatic Admittance to Meetings SHOULD Be Restricted
+#### MS.TEAMS.2.3v1, instructions:
```

Comment:
After reviewing the instructions for MS.TEAMS.2.1v1, MS.TEAMS.2.2v1, and MS.TEAMS.2.3v1, they seem to be the same policy. Even though the policy statements are worded differently and mention different target user groups, the implementation makes it clear that all of these are referencing the "Who can bypass the lobby" setting under the Meeting Policies > Global (Org-wide default) policy.

My recommendation is to combine these three into a single policy which may read something like "****Everyone** SHOULD NOT be admitted automatically in the Global (Org-wide default) meeting policy**" or similar. The term Everyone in this context refers to the setting value of "Everyone" in the drop down box. Any value other than Everyone would be compliant with the policy. There are numerous compliant values that an agency user could select including People in my org and guests, People in my org, trusted orgs and guests, People in my org, and more. ScubaGear would produce a Pass for any of those values, as long as Everyone is not the selected value.
I think this will give flexibility to agencies and they can decide how strict they want to make the setting (as long as they don't allow Everyone to automatically join) for their use cases.

![image](https://github.com/cisagov/ScubaGear/assets/107076927/255aab59-21e2-4c32-92f1-de8f46c05e9c)

Reply:
I get that "Everyone" is an option from the dropdown, but I think "Everyone SHOULD NOT be admitted automatically" is confusing. To me it reads as "No one should be admitted automatically," which isn't what we're going for at all.

Also, "just don't select everyone" isn't actually equivalent to our current policies, as "only organizers and co-organizers" and "people who were invited" aren't options that meet the current MS.TEAMS.2.2v1. I think the rationale for dictating that internal users should be admitted automatically is to reduce "admit fatigue," if users have to manually admit everyone they might be more likely to admit everyone without thought. You could argue that this isn't a big deal, and I could accept that, but I wanted to at least point it out.

Reply:
IMO this group still need three policies may be with modified Notes. 
The first sub-policy that block automatic admittance for anonymous users may not need a custom policy exemption. The second one to allow internal users is to avoid admit fatigue as Alden called above. The third "MAY" policy need better clarification in the notes about custom policy aspect. We can discuss this along with a possible grouping of 1,2,3 and 9.